tests/test-projects/index-ref/tsup.config.js

```diff
@@ -0,0 +1,16 @@
+import { makeConfig } from '@fp-tx/build-tools'
+
+export default makeConfig(
+  {
+    basePath: '.',
```

Comment:
I think we may need to adjust this, the output from tsup in github seems to be building the main project instead of this test dir:

https://github.com/fp-tx/build-tools/actions/runs/7998243402/job/21863451423#step:9:1

Reply:
No, I think that's right. It runs a build before executing the tests (script is `pnpm build && pnpm test ...`) because it is, essentially, an entire new project, with `@fp-tx/build-tools` as a dependency. And the build logs for _that_ aren't actually printed. I can make them print

Reply:
Ah right, cool.  We can add the log in any time, but I'm ready to merge this whenever